ShutterSystem/lightsensor.c

```diff
@@ -0,0 +1,37 @@
+/*

```

Comment:
Dit is waarschijnlijk een test file van je? Deze kan je op zich weer verwijderen neem ik aan. De lightsensor wordt in shutter.c al gelezen